IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.m

```diff
@@ -29,17 +29,70 @@ @implementation MSIDCurrentRequestTelemetry
 
 - (NSString *)telemetryString
 {
-    return @"";
+    return [self serializeCurrentTelemetryString];
 }
 
-- (instancetype)initWithTelemetryString:(__unused NSString *)csvString error:(__unused NSError **)error
+- (instancetype)initWithTelemetryString:(NSString *)telemetryString error:(NSError **)error
 {
     self = [super init];
     if (self)
     {
+        NSError *internalError;
+        [self deserializeCurrentTelemetryString:telemetryString error:&internalError];
         
+        if (internalError) {
+            MSID_LOG_WITH_CTX_PII(MSIDLogLevelError, nil, @"Failed to initialize server telemetry, error: %@", MSID_PII_LOG_MASKABLE(internalError));
+            if (error) *error = internalError;
+            return nil;
+        }
     }
     return self;
 }
 
+#pragma mark - Private
+
+-(NSString *)serializeCurrentTelemetryString
+{
+    int forceRefreshValue = (self.forceRefresh ? 1 : 0);
```

Comment:
this can be simplified to: 
`int forceRefreshValue = self.forceRefresh ? 1 : 0;`

Reply:
This line was changed in the update with the SerializedItems object